<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputPrompt</class>
 <widget class="QDialog" name="InputPrompt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>280</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="loopButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>351</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When you reach the final file, pressing right will bring you to the first.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Loop files after reaching end</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="recurseButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>381</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Instead of looking just into the parent directory, take files recursively from underlying directories. Note that this does not handle duplicated filenames if they happen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Recursively traverse directories</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="inputPath">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Path to the folder containing images that you want to filter.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="outputPath">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Path to the directory in which you save the images that you selected.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imagesSelect">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Opens a file browser.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>ImagesFolder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outputFolder">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Opens a file browser.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>OutputFolder</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InputPrompt</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InputPrompt</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
